src/utils/DockerUtil.js

```diff
@@ -20,20 +21,25 @@ export default {
       throw new Error('Falsy ip or name passed to docker client setup');
     }
 
-    let certDir = path.join(util.home(), '.docker/machine/machines/', name);
-    if (!fs.existsSync(certDir)) {
-      throw new Error('Certificate directory does not exist');
-    }
+    if (util.isLinux()) {
+      this.host = 'localhost';
+      this.client = new dockerode({socketPath: '/var/run/docker.sock'});
```

Comment:
As described in the wiki page [Install Docker](https://github.com/kitematic/kitematic/wiki/Early-Linux-Support#install-docker), you can add your user to the docker group so that you don't need to run Kitematic as root.